src/containers/execution-environment-detail/execution_environment_detail_activities.tsx

```diff
@@ -128,23 +128,29 @@ class ExecutionEnvironmentDetailActivities extends React.Component<
                   if (!!removed) {
                     activityDescription = (
                       <React.Fragment>
-                        <TagLink tag={action.tag_name} /> was moved to{' '}
-                        <ShaLink digest={action.manifest_digest} /> from
-                        <ShaLink digest={removed.manifest_digest} />
+                        <Trans>
+                          <TagLink tag={action.tag_name} /> was moved to{' '}
+                          <ShaLink digest={action.manifest_digest} /> from
```

Comment:
```suggestion
                          <ShaLink digest={action.manifest_digest} /> from{' '}
```

(otherwise `npm run gettext:extract` extracts this as `"<0/> was moved to <1/> from<2/>"`)

Reply:
Ok.